templates/include/telemetry/processData.1.schema.json

```diff
@@ -1,24 +1,17 @@
-"processData": {
   "type": "object",
   "properties": {
     "scalars": {
       "type": "object",
       "additionalProperties": {
-        "scalar": {
-          "type": [ "integer", "string", "boolean" ]
-        }
+        "type": [ "integer", "string", "boolean" ]
```

Comment:
Should scalars be a template since it is repeated in multiple places?

Reply:
Yes, I filed #135 to follow up on this.